src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+class Duke {
+
+    public static void main(String[] args) throws DukeException{
+        Scanner sc = new Scanner(System.in);
+        FileClass fc = new FileClass(); //file class 
+        String home = System.getProperty("user.home"); //home directory
+
+        String filePath = home + "/Desktop/ip/duke.txt";
+        System.out.println(filePath);
+        fc.createFile(filePath);
+
+        ArrayList<Task> taskArray = new ArrayList<Task>();
+        ArrayList<String> stringArray = new ArrayList<String>();
+
+        String beginning = "Hello! I'm Duke\n" +
+                           "What can I do for you?";
+        System.out.println(beginning);
+
+        String input = sc.nextLine();
+
+        while (!input.equals("bye")) {
+            String[] checkType = input.split(" ");
+            String messageTask = "Got it. I've added this task: \n";
+
+            try {
+                if (input.equals("list")) {
+
+                    if (taskArray.isEmpty()) {
+                        System.out.println("The list is empty");
+                    } else {
+                        System.out.println("Here are the tasks in your list:");
+                        for (int i = 0; i < taskArray.size(); i++) {
+                            String index = String.valueOf(i + 1);
+                            System.out.println(index + "." + taskArray.get(i));
+                        }
+                    }
+                } else if (checkType[0].equals("mark") || checkType[0].equals("unmark")) { //this is to check whether it goes through marking
```

Comment:
I think this line has more than the 120 character line limit allowed by the coding standard. Maybe you could add a line break to improve readability